README.md

```diff
@@ -32,7 +32,7 @@ After you're finished please remove all the comments and instructions!
 
 </div>
 
-<!-- TABLE OF CONTENTS -->
+<!-- TABLE OF CONTENTS -->....
 
```

Comment:
- [ ] kindly ensure your MIT license link is working.

Reply:
Thanks it is fixed